source/guides/guides/parallelization.md

```diff
@@ -92,10 +121,127 @@ Machine 1 (Total of 39s)                  Machine 2 (Total of 38s)
 
 Parallelizing our tests across only 1 extra machine saved us about 2 minutes of our already small runtime. This time saved adds up on test runs that could be up to 30 minutes in length.
 
-# Visualizing parallelization in the Dashboard
+# Grouping test runs
+
+Multiple {% url "`cypress run`" command-line#cypress-run %} calls can be labeled and associated to a **single** test run by passing in the {% url "`--group <name>` flag" command-line#cypress-run-group-lt-name-gt %}, where `name` is an arbitrary reference label. The group name must also be unique within the associated test run.
+
+{% note info %}
+For multiple runs to be grouped into a single run, it is required for CI machines to share a common CI build ID environment variable. Typically these CI machines will run in parallel or within the same build workflow or pipeline, but **it is not required to use Cypress parallelization to group runs**. Grouping of runs can be utilized independently of Cypress parallelization.
+{% endnote %}
+
+## Grouping by browser
+
+You can test your application against different browsers and view the results under a single run within the dashboard. Group names could simply be the name of the browser being tested:
+
+- The first group can called `electron`. *Electron is the default browser used in Cypress runs*.
+
+  ```shell
+  cypress run --record --group electron
+  ```
+
+- The second group can be called `chrome`.
+
+  ```shell
+  cypress run --record --group chrome --browser chrome
+  ```
+
+## Grouping with parallelization
+
+We also have the power of Cypress parallelization, so we could speed up testing of any desired group. For the sake of demonstration, let's run a group to test against Chrome with 2 machines, one group to test against Electron with 4 machines, and another group to test against Electron again, but only with one machine:
+
+```shell
+cypress run --record --group 1x-electron
+```
+
+```shell
+cypress run --record --group 2x-chrome --browser chrome
+```
+
+```shell
+cypress run --record --group 4x-electron
+```
+
+The `1x`, `2x`, `4x` group prefix used here is simply an adopted convention to indicate the level of parallelism for each run, and *is not required or essential*.
+
+{% note info %}
+Please do take into account the number machines dedicated for each `cypress run` call is based on your CI configuration for the project.
```

Comment:
`Grammar`

`the number machines` ➡️ `the number of machines`

But, consider deleting the opening preamble in pursuit of a leaner sentence: you've already drawn attention to the sentence by putting it into a `note`. Yeah?

```md
The number of machines dedicated for each `cypress run` call is based on your CI configuration for the project.
```